semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/error/ErrorSemanticConventionUtils.java

```diff
@@ -0,0 +1,50 @@
+package org.hypertrace.semantic.convention.utils.error;
+
+import java.util.Arrays;
+import org.hypertrace.core.datamodel.Event;
+import org.hypertrace.core.datamodel.shared.SpanAttributeUtils;
+import org.hypertrace.core.span.constants.RawSpanConstants;
+import org.hypertrace.core.span.constants.v1.Error;
+import org.hypertrace.core.span.constants.v1.OTSpanTag;
+import org.hypertrace.traceenricher.enrichedspan.constants.v1.ErrorMetrics;
+
+/**
+ * Utility class for fetching error related attributes
+ */
+public class ErrorSemanticConventionUtils {
+
+  private static final String OTEL_EXCEPTION_TYPE = OTelErrorSemanticConventions.EXCEPTION_TYPE.getValue();
+  private static final String OTEL_EXCEPTION_MESSAGE = OTelErrorSemanticConventions.EXCEPTION_MESSAGE.getValue();
+  private static final String OTEL_EXCEPTION_STACK_TRACE = OTelErrorSemanticConventions.EXCEPTION_STACKTRACE.getValue();
+
+  private static final String[] EXCEPTION_ATTRIBUTES = {
+      RawSpanConstants.getValue(Error.ERROR_ERROR),
+      RawSpanConstants.getValue(OTSpanTag.OT_SPAN_TAG_ERROR),
+      OTEL_EXCEPTION_TYPE
```

Comment:
I think, we should add `OTEL_EXCEPTION_MESSAGE` this as well into the list. Based on this - https://github.com/open-telemetry/opentelemetry-specification/blob/master/semantic_conventions/trace/exception.yaml - it looks like` any_of`

Reply:
Sure, I will add. 
I am assuming this particular attribute should contribute to `ERROR_METRICS_EXCEPTION_COUNT` ?

Reply:
Added